crypto/pem/pvkfmt.c

```diff
@@ -1062,7 +1063,7 @@ static int i2b_PVK(unsigned char **out, const EVP_PKEY *pk, int enclevel,
             goto error;
         if ((rc4 = EVP_CIPHER_fetch(libctx, "RC4", propq)) == NULL)
             goto error;
-        if (enclevel == 1)
+        if (encrypt == 1)
```

Comment:
Hmm this line is a bit suss.. Does that mean encrypt could or been set to anything other than zero before, but then only do this line if it is 1? (You sure it was a boolean :))

Reply:
Hmm. The doc states either 0 or 1 so I'll go with that :)

Reply:
But that would make this test pointless..

Reply:
This change has been reversed.